docs/deployment/uwsgi_cookiecutter_nginx.rst

```diff
@@ -0,0 +1,233 @@
+.. _uwsgi_tutorial:
+
+Running a Cookiecutter :app:`Pyramid` Application under :app:`uWSGI` and :app:`Nginx`
+=====================================================================================
+
+``uWSGI`` is a software application for building hosting services.
+It is named after the Web Server Gateway Interface (the WSGI specification
+to which many Python web frameworks conform).
+
+This guide will outline broad steps that can be used to get a Cookiecutter
+:app:`Pyramid` application running under Nginx via ``uWSGI``.  This particular
+tutorial was developed under Ubuntu 18.04, but the instructions should be
+largely the same for all systems, delta specific path information for commands
+and files.
+
+
+#.  Install prerequisites
+
+    .. code-block:: bash
+
+       $ sudo apt install -y uwsgi-core uwsgi-plugin-python3 python3-cookiecutter \
+                             python3-pip python3-venv nginx
+       $ python3 -m cookiecutter gh:Pylons/pyramid-cookiecutter-starter --checkout master
```

Comment:
Note that if you use `--checkout master`, then you might get the "bleeding edge" version with some sharp edges, and the correlating version of Pyramid. Suggest omitting that flag for safety, unless you really need what `master` branch provides instead of `latest`, the default stable branch.